taxi/forms.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.validators import RegexValidator
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    license_number = forms.CharField(
+        max_length=8,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message="License number must consist"
+                        " of 3 uppercase letters followed by 5 digits."
+            ),
+        ]
+    )
+    first_name = forms.CharField(max_length=30)
+    last_name = forms.CharField(max_length=30)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
```

Comment:
you already have `max_length=8` on a form so this method is unnecessary. What you could do is to add MaxValueValidator to the validators attribute to make your code better structured